db/new_seeds/base/add_schools_for_cohortless_testing.rb

```diff
@@ -6,7 +6,9 @@ def padded_urn(num)
 
 cohort_22 = Cohort.find_by(start_year: 2022) || FactoryBot.create(:seed_cohort, start_year: 2022)
 cohort_23 = Cohort.find_by(start_year: 2023) || FactoryBot.create(:seed_cohort, start_year: 2023)
+cohort_24 = Cohort.find_by(start_year: 2024) || FactoryBot.create(:seed_cohort, start_year: 2024)
```

Comment:
I wonder if instead of adding cohorts here we can use previous and current cohort? and amend the naming to match that? cc @tonyheadford any thoughts on this?

Reply:
Good idea